scripts/install.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+set -e
+
+DOWNLOAD_URL="https://github.com/redhat-developer/ocdev/releases/download/v0.0.1"
+OCDEV_VERSION="latest"
+BINARY_NAME="ocdev"
+CANCELLATION_TIMEOUT=15
```

Comment:
what does this delay do? can we have a comment explaining it?

Reply:
added